packages/discord.js/src/structures/Channel.js

```diff
@@ -27,7 +27,8 @@ class Channel extends Base {
      * The type of the channel
```

Comment:
The Channel type has not been changed (I can't reach it)

Reply:
Not sure what you mean here, do you mind elaborating 😀

Reply:
![image](https://user-images.githubusercontent.com/42935195/149996453-0de4d9d6-9513-4a44-8fdf-dd7c5b39a527.png)

Reply:
ah ok I was looking below not above